test/FormBodyTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Amp\Artax\Test;
+
+use Amp\Artax\FormBody;
+use Amp\ByteStream\Message;
+use Amp\PHPUnit\TestCase;
+use function Amp\Promise\wait;
+
+class FormBodyTest extends TestCase {
+    public function testUrlEncoded() {
+        $body = new FormBody();
+        $body->addFields([
+            'a' => 'a',
+        ], 'application/json');
+        $body->addField('b', 'b', 'application/json');
+        $body->addField('c', 'c', '');
+        $body->addField('d', 'd');
+        $body->addFields([
+            'e' => 'e',
+        ], '');
+        $body->addFields([
+            'f' => 'f'
+        ]);
+        $content = wait($body->createBodyStream()->read());
```

Comment:
I think this should be `$content = wait(new Message($body->createBodyStream()));`